pages/mods.vue

```diff
@@ -504,11 +540,16 @@ export default {
         ]
 
         if (this.query.length > 0) {
+          console.log(this.query)
```

Comment:
does this debug line need to be here?

Reply:
Nope, my bad. Should be fixed with 62c3404b8cf8cd78b29f3b20a661ddf992cceb25